src/components/approve/ApproveLayout.tsx

```diff
@@ -23,19 +25,28 @@ export const ApproveLayout = ({
     showHigherCustomFeeBanner?: boolean;
     setShowHigherCustomFeeBanner?: (value: boolean) => void;
     hasHigherCustomFee?: number | null;
+    hasLowerCustomFee?: number | null;
     wallet?: Contracts.IReadWriteWallet;
     containerClassName?: string;
 }) => {
+    const hasCustomFee = hasHigherCustomFee || hasLowerCustomFee;
+    const customFeeState = hasCustomFee
+        ? hasHigherCustomFee
+            ? constants.FEE_HIGHER
+            : constants.FEE_LOWER
+        : null;
```

Comment:
this should be a translation, not a constant. Otherwise it will output `higher` and `lower` in english at all times